libs/design-system/logos/apps/src/lib/logos-apps.component.html

```diff
@@ -0,0 +1,13 @@
+<a class="hra-logo" href="https://humanatlas.io" target="_blank" rel="noopener noreferrer">
```

Comment:
Add `aria-label="Visit Human Reference Atlas"`
Accessibility concatenates the image alt and text to create a label for the link which means it is currently showing
`Human Reference Atlas home Human Reference Atlas` which is less than ideal